silver/tests/factories.py

```diff
@@ -247,6 +247,23 @@ class Meta:
     customer = factory.SubFactory(CustomerFactory)
 
 
+class ProformaTransactionFactory(factory.django.DjangoModelFactory):
```

Comment:
Maybe use `TransactionFactory` like this: `TransactionFactory.create(invoice=None)`. Might need to make it work first.